internal/pkg/util/paths/resolve_test.go

```diff
@@ -97,15 +97,30 @@ func TestSoLinks(t *testing.T) {
 
 func TestPaths(t *testing.T) {
 	// Very naive sanity test. Check we can find one lib and one binary without error
```

Comment:
Extend this comment to include a configuration file.

Reply:
thanks! I refined the comment line.